### ISSUE TITLE
[CR127] Icons in the download bubble & Permission prompts should use Nala icons

### ISSUE THREAD

**Issue by MadhaviSeelam** [brave/brave-browser#39395]
### Description

Found while testing https://github.com/brave/brave-browser/issues/39363.  Download bubbles use Nala icons.  Permission prompts titles, text, icons and spacing need fix

### Steps to reproduce

1. Install 1.69.51
2. launch Brave
3. try to download a file (malicious and/or regular)
4. visit a site that asks for permissions

### Actual result

example | example | example 
------ | ----- | -----
![image (1)](https://github.com/brave/brave-browser/assets/98358127/baa16fbb-ccea-43c8-a41b-0faad32a84b5)|![image (2)](https://github.com/brave/brave-browser/assets/98358127/5d14d7a9-91cc-4189-88fe-0c7a52d8b67b)|![image](https://github.com/brave/brave-browser/assets/98358127/ba3599b9-f458-45d9-8f09-d7f814b9ff74)

### Expected result

Should be Nala icons 

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.69.51 Chromium: 127.0.6533.17 (Official Build) nightly (64-bit)
-- | --
Revision | 8f43b9af539c59840245a61887d5447733d76af5
OS | Windows 11 Version 23H2 (Build 22631.3810)

### Channel information

- [ ] release (stable)
- [ ] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

@rebron @aguscruiz 
cc: @qa

**Comment by simonhong** [comment]
checking now.

**Comment by simonhong** [comment]
In the download bubble, it's ok if triangle warning is replaced with ours?
<img width="361" alt="image" src="https://github.com/brave/brave-browser/assets/6786187/accbaf13-66ee-40ed-803f-e278a66b163e">

**Comment by aguscruiz** [comment]
No, we should also change the red one. Let's use 'disable-outline' for that one. In red

**Comment by simonhong** [comment]
Upstream's icon is dangerous_chrome_refresh.icon
When it's replaced with leo_disable_outline, our icon is set in download bubble like this.
<img width="354" alt="Screenshot 2024-07-10 at 8 51 10 PM" src="https://github.com/brave/brave-browser/assets/6786187/1da8d340-9039-48d2-b0e9-71ea977435db">

but also it's shared with omnibox like this.
<img width="522" alt="image" src="https://github.com/brave/brave-browser/assets/6786187/544a1aa0-2c30-44b1-aa87-d7565f688343">

Maybe we only want to change download bubble's one?

**Comment by aguscruiz** [comment]
Ok, for the sake of making it closer to chromium, I created a new icon `dangerous-filled`. 
You'll need an updated set from Nala. 

Let's use that one in both the download bar and url bar. What do you think?

![image](https://github.com/brave/brave-browser/assets/15734630/28681c5a-76ef-486b-af76-6a0ba99a4812)

**Comment by rebron** [comment]
@simonhong Is this fixed with https://github.com/brave/brave-core/pull/24582 per your comment above?

**Comment by simonhong** [comment]
> @simonhong Is this fixed with [brave/brave-core#24582](https://github.com/brave/brave-core/pull/24582) per your comment above?

Only triangle warning icon is updated.
Other red danger could be updated after nala bump.

**Comment by rebron** [comment]
Discussed with @aguscruiz. Closing as stale.
